lib/migrator.js

```diff
@@ -180,6 +219,12 @@ Migrator.prototype = {
             return;
           }
 
+          if (self.internals.check) {
+            log.info('Migrations to run:', toRun.map(migration => migration.name));
+            callback(null);
```

Comment:
It is probably more worthwhile to not only log it, but to deliver the actual list for the programmatic mode.